src/main/resources/registered_tokens/babylonnet.csv

```diff
@@ -0,0 +1,2 @@
+id|name|standard|account_id
```

Comment:
Would it be possible to use commas as delimiters here? That would make the CSV files nice and searchable on GitHub?

Reply:
will do

Reply:
As @piotrkosecki pointed out below, we might want to integrate this in his own, more general, open PR.

Reply:
will surely do a separate PR for this, right after all the currently pending are settled, to reduce the impact